doc/RecordingControl.xml

```diff
@@ -1684,8 +1766,220 @@ SetRecordingJobMode(JobToken, Active)
+        If a <literal>Postfix</literal> is configured, the device shall include <literal>postfix</literal> in <literal>lead</literal> and replace it with the configured <literal>Postfix</literal>.
+        The device shall replace <literal>date</literal> and <literal>time</literal> respectively with the date and time portion of the segment start in Universal Coordinated Time (UTC), following the ABNF rules.
+        The device shall replace <literal>counter</literal> with a counter reset to 0 for a new span and incremented by 1 when closing segments of the underlying span.
+        The device shall replace <literal>extension</literal> with a value defined by the recording formats in the following sections.
+      </para>
+      <para>
+        Note: <literal>date</literal> is conformant to the extended date format defined in ISO 8601-1.
+        <literal>time</literal> is not conformant to the extended time format defined in ISO 8601-1, as it replaces <literal>":"</literal> with <literal>"-"</literal> compared to that format.
+      </para>
+    </section>
+    <section>
+      <title>Span end objects</title>
+      <para>
+        To store the end time of of a span, the device should store an additional empty object after closing a span.
+        The device shall generate the object key for this object according to the ABNF <literal>key</literal> rule and replacement rules defined in <xref linkend="_refCloudRecordingSegmentObjects" />.
+        The device shall replace <literal>date</literal> and <literal>time</literal> respectively with the date and time portion of the span end in Universal Coordinated Time (UTC), following the ABNF rules.
+        The device shall replace <literal>counter</literal> by a value equal to the counter of the last segments of the span plus 1.
+        The device shall replace <literal>extension</literal> with a value defined by the recording formats in the following sections.
+      </para>
+    </section>
+    <section>
+      <title>Other objects</title>
+      <para>
+        The device may store additional objects with object keys starting with <literal>lead "/"</literal> according to the ABNF rules and replacement rules defined in <xref linkend="_refCloudRecordingSegmentObjects" />, as long as their object keys do not end in an extension defined in this annex.
+      </para>
+    </section>
+    <section>
+      <title>Recording format MP4</title>
+      <para>
+        For the recording format <literal>MP4</literal>, the recording information shall be written as multiplexed tracks into a single segment.
+        Each segment shall contain zero or one tracks with video data, zero or one tracks with audio data, and zero or one tracks with metadata.
+        The device shall generate the object key for each segment with the <literal>extension</literal> <literal>"mp4"</literal>.
+      </para>
+      <para>
+        The device shall upload each segment object with the content type <literal>video/mp4</literal>.
```

Comment:
"end object" files should probably have a distinct content type (avoid "video/mp4"). I feel that it should be predefined here. Idem for CMAF.

Reply:
Would that benefit clients? Would you define a compeltely custom Content-Type?

Reply:
Nevermind, I thought that the content-type was mandatory for some Cloud Storage providers. It seems that it's not the case.